internal/commands/root.go

```diff
@@ -0,0 +1,69 @@
+package commands
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+)
+
+const (
+	defaultConfigFilename = "policy-hub"
+	envPrefix             = "POLICYHUB"
+)
+
+// NewRootCommand returns the root command for the policy-hub CLI
+func NewRootCommand() *cobra.Command {
+	rootCmd := &cobra.Command{
+		Use:   "policy-hub <command>",
+		Short: "Search through Rego policies",
+		PersistentPreRunE: func(cmd *cobra.Command, _ []string) error {
+			return initializeConfig(cmd)
+		},
+		SilenceUsage: true,
+	}
+
+	return rootCmd
+}
+
+// initializeConfig initializes viper config and binds viper to the cobra flags.
```

Comment:
We should decide if we are going to comment on both exported and un-exported functions. 
I wont strongly advocate for either, we should just strive for consistency across the codebase.